### ISSUE TITLE
Changing units -- need to support two use cases

### ISSUE THREAD

**Issue by hcferguson** [spacetelescope/specviz#132]
There are two use cases for changing units:

1. The units I have on the spectrum are correct, and I want to change what is viewed on the plot.
2. The units I have on the spectrum are incorrect and I need to correct them.

I believe we are supporting (1) right now but not (2).

When I read in example_spec_ascii.txt, it defaults to Angstrom because no units are provided. However when I try to change to micron, it is changing the numbers, rather than just axis label.

**Comment by pllim** [comment]
> 2. The units I have on the spectrum are incorrect and I need to correct them.

IMHO, the real solution for this is to correct the data file itself. It should not be the job of a GUI tool to fix the file unless it can also write the corrected file back out.

**Comment by hcferguson** [comment]
@plim I don't really disagree...but we had decided to default to Angstroms when no units were specified. So we should either throw an error there (which I don't think is particularly user friendly), or maybe open a dialog box when the tool discovers there are no units and prompt the user to enter them when reading in the file?

**Comment by pllim** [comment]
@hcferguson , it is a lot easier for the user to just modify his/her own YAML file (or the actual data file) and specify the correct unit. I just don't see the point of a GUI feature for this. But we can agree to disagree.

**Comment by nmearl** [comment]
I have to agree with @pllim. I don't think we should implement a gui means of correcting a data set's units. This can be done by the user by editing either yaml file, or the data directly.

**Comment by hcferguson** [comment]
Can we pop up an error when no units are supplied that points them to the documentation?

**Comment by nmearl** [comment]
This is related to #219, as that issue provides a solution for defining correct units.

**Comment by ibusko** [comment]
Given @nmearl 's comment above, and the spectrum ingestion wizard that was implemented after these discussions took place, I believe this issue can be closed now.
